AstronomyTests/FetchPhotoOperationTests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  FetchPhotoOperationTests.swift
+//  AstronomyTests
+//
+//  Created by Bradley Yin on 9/16/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import Foundation
+import XCTest
+@testable import Astronomy
+
+class FetchPhotoOperationTests: XCTestCase {
+    
+    func testStartOperation() {
+        var photos : [MarsPhotoReference] = []
+        let data = validSol1JSON!
+        let jsonDecoder = MarsPhotoReference.jsonDecoder
+        let decodedObject = try! jsonDecoder.decode([String : [MarsPhotoReference]].self, from: data)
+        photos = decodedObject["photos"]!
+        
+        let fetchPhotoOperation = FetchPhotoOperation(photoReference: photos[0])
+        fetchPhotoOperation.start()
+        
+        sleep(5)
```

Comment:
wait for expectations!

<details>
<summary>spoiler method</summary>

    func testStartOperation() {
        var photos : [MarsPhotoReference] = []
        let data = validSol1JSON!
        let jsonDecoder = MarsPhotoReference.jsonDecoder
        let decodedObject = try! jsonDecoder.decode([String : [MarsPhotoReference]].self, from: data)
        photos = decodedObject["photos"]!
        
        let fetchPhotoOperation = FetchPhotoOperation(photoReference: photos[0])
        fetchPhotoOperation.start()

		let expecting = expectation(description: "test")
		DispatchQueue.global().async {
			var run = true
			while run {
				if fetchPhotoOperation.state == .isFinished {
					expecting.fulfill()
					run = false
				}
			}
		}

		waitForExpectations(timeout: 10) { (error) in
			if let error = error {
				XCTFail("Timed out waiting for an expectation: \(error)")
			}
		}

        XCTAssertTrue(fetchPhotoOperation.image != nil)
    }

</details>